README.md

```diff
@@ -1,24 +1,16 @@
-# Lily watchdog
+# SaffronStatus
 
-This is a simple bot designed to monitor the status of [LilyBot](https://github.com/IrisShaders/LilyBot).
+This is a simple bot designed to monitor the online status of a bot
 
-When the bot is offline for more than 2 consecutive minutes, a notification will be posted to the selected channel, 
-provided in the `.env` file. When the bot is then online again, a new notification will be posted and a downtime summary
+When a bot is offline for more that your specified number of minutes, a notification will be posted to the selected channel, 
```

Comment:
I recognize there's only one command with a two subcommands but it'd be nice to have simple docs. It can just be a `adding a bot to watch` and a `removing a bot to watch` section. I can do this writing if you'd like.